qays/style.css

```diff
@@ -0,0 +1,274 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+
+}
+
+header{
+    flex-direction: row;
+    margin: 16px;
+
+}
+header nav{
+    display: flex;
+    justify-content: space-between;
+}
+
+
+img[alt="logo"]{
+    height: 80px ;
+}
+nav ul{
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+}
+nav ul li a{
+ margin-left: 10px; 
+ text-decoration: none;        
+}
+
+.column-about{
```

Comment:
these aren't necessary since they will already be ordered according to there order in HTML